### PR TITLE
Davidsymonds

### PR DESCRIPTION
**Pull request by ricjhill**
This is the story about how my stepfather became a SW Tester in 1990. I hope you like it.

### CODE REVIEW

File: davidsymonds/David_Symonds.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# A Taxing Situation
+
+## How the UK poll tax lead to a career in software testing.
+
+Wikipedia describes The Community Charge commonly known as the poll tax as “a system of taxation introduced by Margaret Thatcher's government in replacement of domestic rates in Scotland from 1989, prior to its introduction in England and Wales from 1990. It provided for a single flat-rate per-capita tax on every adult, at a rate set by the local authority. The charge was replaced by Council Tax in 1993, two years after its abolition was announced.”  Its introduction lead to riots, punk rock songs, the fall of the one of the UKs most successful Prime Ministers and my move into Software Testing.
```
Comment (flynnbops) [inline davidsymonds/David_Symonds.md]: Punchy opening. 

Is it worth linking to the wikipedia page as a source?

File: davidsymonds/David_Symonds.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+I had accepted a job with British Telecom (BT) as a test manager on a poll tax system for 14 local authorities. At that time there were only two telecom companies in the UK. The largest BT was contracted to provide a system to package and print the poll tax bills, payment books and ancillary documents that local authorities sent out each year. The system also printed the individual Local Authorities coats of arms on the documents. This project had started in the previous October. It was running late and the documents had to be produced before the beginning of April. I and my six colleagues had to get the project back on track. To this end we worked the extended hours expected nowadays from a Start-up.
```
Comment (flynnbops) [inline davidsymonds/David_Symonds.md]: Does "This project had started in the previous October" work a bit more consicely as `This project started the previous October`?

File: davidsymonds/David_Symonds.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+1990 was sometime before the internet was useful for troubleshooting this type of problem, so as was my habit I referred to my personal engineering notes that I had kept since 1968. Here I found that a process existed within Norton Utilities for retrieving deleted files. As luck would have it, there was a copy of Norton on site. We retrieved all of the data in about three hours.
+
+In early March 1990 we moved the entire operation from Brighton to Woking where BT had a major printing, sorting and distribution centre. Towards the end of March we were ready to receive and process the local authorities’ data.
+
+This process involved printing each local authorities bills direct debit instructions and payment books together with the supporting documents.
```
Comment (flynnbops) [inline davidsymonds/David_Symonds.md]: Does it need a comma? `This process involved printing each local authorities bills, direct debit instructions and payment books together with the supporting documents.`

File: davidsymonds/David_Symonds.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+In the distribution centre were four high speed laser printers that worked at 180 A4 pages per minute. Payment books were guillotined stapled and assembled ready to go into the hoppers of the collating machines. These machines assembled the documents for an individual, enveloped them and sorted the envelopes into walking order for the postman.
```
Comment (flynnbops) [inline davidsymonds/David_Symonds.md]: Does it need a comma? `Payment books were guillotined, stapled ...`.

File: davidsymonds/David_Symonds.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+When I got back to Woking I discussed with my programmer what was required in order to change the block size to its maximum 32768 which would allow us to read and write the maximum number of records possible. I then contacted the ops manager at Berkshire to cost the amendment to the file block size on the disc’s the quote was £800. I ran the costing by the systems manager and product director and we all agreed that this would be a significant performance enhancement. It took one day to amend the program’s and Berkshire completed their changes on the same day.We performed some comparisons tests and the results appeared very favourable reducing the time it took to run through the full system from over 20 hours to under 5. When the 1991 New Year poll tax took place we saved over 2010 hours in the processing time which at the time we paid Berkshire £120 per hour making a cost saving in access of £25,000.
+
+I am retired now and if I were to give advice to someone in today's software testing industry  today it would be to "Use notes memories and experience  to help others correcting faults. A testers role is to ensure that  the product is  fit for purpose."
```
Comment (flynnbops) [inline davidsymonds/David_Symonds.md]: Another comma? `Use notes, memories and experience`

File: davidsymonds/David_Symonds.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Biography
+
+I joined Exide Battery computer operations as a trainee operator in 1968. Over the next 9 years I moved up the ladder to Chief Operator then into programming and systems. In 1979 I moved to Southend as a Senior Systems Analyst. Two years later I transferred to the adjoining council of Rochford as Deputy Director of Computer Services. I held this position for eight years until the department was outsourced. I was then faced with either joining the outsourcing company or being made redundant. I chose redundancy as I had been thinking about working for myself. My dilemma became which area of I.T. I should specialise in. This was resolved when I was invited by International Computer Limited (ICL) to deliver a disaster recovery project as the project manager. January the following year in 1990 I accepted a job with British Telecom (BT) as a Test Manager.
```
Comment (flynnbops) [inline davidsymonds/David_Symonds.md]: Are we not including a name?

**Review by flynnbops — APPROVED** [review]
Great story. Some minor suggestions from me

File: davidsymonds/David_Symonds.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# A Taxing Situation
+
+## How the UK poll tax lead to a career in software testing.
```
Comment (LouiseWoodhams24) [inline davidsymonds/David_Symonds.md]: Should this be led rather than lead?

File: davidsymonds/David_Symonds.md (hunk 1 above)
Comment (LouiseWoodhams24) [inline davidsymonds/David_Symonds.md]: Another lead here that should be led

**Review by LouiseWoodhams24 — COMMENTED** [review]
Really lovely story, lots of detail and with good advice for current testers. Please Tell Your Step Dad he is awesome

File: davidsymonds/David_Symonds.md (hunk 2 above)
Comment (ricjhill) [reply davidsymonds/David_Symonds.md]: Yes. I would have written that but he was telling a story to me and I'd like to keep those archaic bits of grammar.

File: davidsymonds/David_Symonds.md (hunk 3 above)
Comment (ricjhill) [reply davidsymonds/David_Symonds.md]: You are right it does.

File: davidsymonds/David_Symonds.md (hunk 4 above)
Comment (ricjhill) [reply davidsymonds/David_Symonds.md]: Again yes it does

File: davidsymonds/David_Symonds.md (hunk 1 above)
Comment (ricjhill) [reply davidsymonds/David_Symonds.md]: corrected

File: davidsymonds/David_Symonds.md (hunk 7 above)
Comment (ricjhill) [reply davidsymonds/David_Symonds.md]: Yes, it should.  That is my mistake too not his

File: davidsymonds/David_Symonds.md (hunk 6 above)
Comment (ricjhill) [reply davidsymonds/David_Symonds.md]: added

File: davidsymonds/David_Symonds.md [hunk 8]
```diff
@@ -1,37 +1,38 @@
...
+When I got back to Woking I discussed with my programmer what was required in order to change the block size to its maximum 32768 which would allow us to read and write the maximum number of records possible. I then contacted the ops manager at Berkshire to cost the amendment to the file block size on the disc’s. The quote was £800. I ran the costing by the systems manager and product director and we all agreed that this would be a significant performance enhancement. It took one day to amend the programs and Berkshire completed their changes on the same day.We performed some comparisons tests and the results appeared very favourable, reducing the time it took to run through the full system from over 20 hours to under 5. When the 1991 New Year poll tax took place we saved over 2010 hours in the processing time, which at the time we paid Berkshire £120 per hour making a cost saving in access of £25,000.
 
-## Biography
+I am retired now and if I were to give advice to someone in today's software testing industry, it would be to: "Use notes, memories and experience  to help others correcting faults. A testers role is to ensure that  the product is  fit for purpose."
```
Comment (LouiseWoodhams24) [inline davidsymonds/David_Symonds.md:27]: there is a double space I think between experience and to

**Review by LouiseWoodhams24 — COMMENTED** [review]
Noticed a double space on line 27. Love the resource section, that computer is super cute!

**Pull request merged**